docs/api-reference/next.config.js/exportPathMap.md

```diff
@@ -79,6 +79,8 @@ module.exports = {
 next export -o outdir
 ```
 
+> **Warning**: Using `exportPathMap` for defining routes with any `getStaticPaths` powered page is now ignored and gets overridden. We recommend not to use them together.
```

Comment:
This is confusing - is "now ignored" - so what exactly changed? And at what version?

If we don't recommend to use together, can we provide more clarity? 

> **Note:** If you are using `getStaticPaths` to define routes, you should _not_ use `exportPathMap`.

Is it really just that?